### PR TITLE
Save a copy of design file to a temporary path

### PR DESCRIPTION
**Pull request by borulday**
Users will be able to export design files with no need to save them specifically.

### CODE REVIEW

File: Zeplin.sketchplugin/Contents/Sketch/utils.cocoascript [hunk 1]
```diff
@@ -1,35 +1,52 @@
...
+
+var temporaryPathForDesignFile = function (context) {
+    var doc = context.document;
+    var path = temporaryPath();
+
+    [doc showMessage:@"Saving document…"];
+    [doc saveToURL:[NSURL fileURLWithPath: path] ofType:@"com.bohemiancoding.sketch.drawing.single" forSaveOperation:NSSaveToOperation delegate:nil didSaveSelector:nil contextInfo:nil];
+
+    var fileManager = [NSFileManager defaultManager];
+    var stopTime = [NSDate dateWithTimeIntervalSinceNow:30];
```
Comment (borulday) [inline Zeplin.sketchplugin/Contents/Sketch/utils.cocoascript:32]: wdyt about this interval value?

Comment (yigitcanyurtsever) [reply Zeplin.sketchplugin/Contents/Sketch/utils.cocoascript:32]: I think 30 secs should be more than enough. 👍

File: Zeplin.sketchplugin/Contents/Sketch/utils.cocoascript [hunk 2]
```diff
@@ -90,7 +107,8 @@ var defaultDirectives = function(context, path) {
     var sketchmigratePath = [[NSBundle mainBundle] pathForResource:@"sketchmigrate" ofType:nil inDirectory:@"sketchtool/bin"];
 
     var directives = [NSMutableDictionary dictionary];
-    [directives setObject:[[doc fileURL] path] forKey:@"path"];
+    [directives setObject:temporaryPath forKey:@"temporaryPath"];
+    [directives setObject:[[doc fileURL] path] forKey:@"path"]; // legacy
```
Comment (yigitcanyurtsever) [inline Zeplin.sketchplugin/Contents/Sketch/utils.cocoascript:111]: We'll always need to send this info to allow users to see which file they exported from. So, I think we shouldn't mark it as 'legacy'.

**Review by yigitcanyurtsever — APPROVED** [review]
LGTM after 'legacy' comment.

**Review by kardeslik — APPROVED** [review]

**Pull request merged**